spec.html

```diff
@@ -10741,6 +10741,8 @@
         IdentifierName but not ReservedWord
     </emu-grammar>
 
+    <emu-note>`yield` and `await` are permitted as |BindingIdentifier| in the grammar, and prohibited with static semantics below, to prohibit ASI in cases such as `let\nawait`.</emu-note>
```

Comment:
s/ASI/automatic semicolon insertion/ and maybe use a `<pre>` block (I haven't checked the generator html, though!).

```html
<pre><code class="javascript">
let
await
0;
</code></pre>
```